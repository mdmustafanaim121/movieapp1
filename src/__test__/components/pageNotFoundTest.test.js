import PageNotFound from '../../component/pageNotFound' 
import renderer from "react-test-renderer";
import React from "react";


describe('Testing Page Not Found Component',()=>{
    it('Should return page not found snapshot',()=>{
        const result = renderer.create(<PageNotFound />).toJSON()
        expect(result).toMatchSnapshot()
    })
})