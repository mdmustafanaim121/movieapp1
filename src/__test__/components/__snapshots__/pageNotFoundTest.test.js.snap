// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Page Not Found Component Should return page not found snapshot 1`] = `
<div
  className="loading"
>
  <img
    alt="No such page exist"
    src="pageNotFound.jpg"
  />
</div>
`;
